changelog.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+Version 0.3.2
+------------
+- Fixed issue https://github.com/remydubois/lsnms/issues/12, by masking scores as well as boxes.
+- Added torch and torchvision as proper dev dependencies
+- Fixed Pillow version to 9.3.0 in dev dependencies because 9.4.0 does not compile on my mbp (see https://github.com/python-pillow/Pillow/issues/6862)
```

Comment:
Hi. It is my hope that this has now been fixed, by https://github.com/python-pillow/Pillow/issues/6862#issuecomment-1384994925